[2022-10-01 19:49:22,005] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: main_dag.create_company_dim_table manual__2022-10-01T19:48:36.834322+00:00 [queued]>
[2022-10-01 19:49:22,021] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: main_dag.create_company_dim_table manual__2022-10-01T19:48:36.834322+00:00 [queued]>
[2022-10-01 19:49:22,021] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-01 19:49:22,021] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-01 19:49:22,022] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-01 19:49:22,055] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_company_dim_table> on 2022-10-01 19:48:36.834322+00:00
[2022-10-01 19:49:22,062] {standard_task_runner.py:52} INFO - Started process 551 to run task
[2022-10-01 19:49:22,072] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'create_company_dim_table', 'manual__2022-10-01T19:48:36.834322+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpmtxakw73', '--error-file', '/tmp/tmpaul5phhd']
[2022-10-01 19:49:22,074] {standard_task_runner.py:80} INFO - Job 60: Subtask create_company_dim_table
[2022-10-01 19:49:22,169] {task_command.py:369} INFO - Running <TaskInstance: main_dag.create_company_dim_table manual__2022-10-01T19:48:36.834322+00:00 [running]> on host cb37a969dddb
[2022-10-01 19:49:22,326] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=main_dag
AIRFLOW_CTX_TASK_ID=create_company_dim_table
AIRFLOW_CTX_EXECUTION_DATE=2022-10-01T19:48:36.834322+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-01T19:48:36.834322+00:00
[2022-10-01 19:49:22,354] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-10-01 19:49:22,361] {dbapi.py:208} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS company_dim_table (
                Company_Full_Name TEXT NOT NULL,
                Shariah TEXT NOT NULL,
                Sector TEXT NOT NULL,
                Company_Key TEXT NOT NULL
            );
        , parameters: None
[2022-10-01 19:49:22,363] {postgres.py:94} INFO - NOTICE:  relation "company_dim_table" already exists, skipping

[2022-10-01 19:49:22,396] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=create_company_dim_table, execution_date=20221001T194836, start_date=20221001T194922, end_date=20221001T194922
[2022-10-01 19:49:22,485] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-01 19:49:22,578] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
